markdown/3.x/en/documentation/tracking-guide.md

```diff
@@ -99,7 +99,13 @@ If you want to prevent the auto requests, you can disable them with the Boolean
    <ff-checkout-tracking-item record-id="10321943" count="1"></ff-checkout-tracking-item>
 </ff-checkout-tracking>
 ```
-If you do so, you can poke around with the child elements and invoke the `checkoutTrackingElement.trackCheckoutItems()` function manually.
+If you want to send tracking request to the different channel, use `channel` attribute
```

Comment:
I am wondering: `to the different channel` vs `to a different channel`. The later option seems to be more accurate in this context. @omikron-fp: what do you make of it?

Reply:
I'd go for a bit more details.

_If you want to send the tracking request to a channel different from the one configured in `ff-communication`, use the `channel` attribute._